cf/data/mixin/deprecations.py

```diff
@@ -155,6 +155,25 @@ def ispartitioned(self):
         """
         _DEPRECATION_ERROR_METHOD("TODODASK")  # pragma: no cover
 
+    def close(self):
+        """Close all files referenced by the data array.
+
+        Deprecated at version TODODASK. All files are now automatically
+        closed when not being accessed.
+
+        """
+        _DEPRECATION_ERROR_METHOD(
+            self,
+            "close",
+            "All files are now automatically closed when not being accessed.",
+            version="TODODASK",
+            removed_at="5.0.0",
+        )  # pragma: no cover
+
+        print("TODODASK - is this still needed/valid? Not needed")
```

Comment:
It's not clear whether this (first) question remains open, since you seem to have answered it yourself already in the next sentence? Just checking there is nothing left to do here, to tidy up or otherwise...

Reply:
Good spot - that deprecated code has now been deleted. Merging ...